src/mean.jl

```diff
@@ -29,99 +9,139 @@ julia> srs = SimpleRandomSample(apisrs;popsize=:fpc);
 
 julia> mean(:enroll, srs)
 1×2 DataFrame
- Row │ mean     sem     
+ Row │ mean     SE     
      │ Float64  Float64 
 ─────┼──────────────────
    1 │  584.61  27.3684
 ```
 """
 function mean(x::Symbol, design::SimpleRandomSample)
+    function se(x::Symbol, design::SimpleRandomSample)
+        variance = design.fpc * Statistics.var(design.data[!, x]) / design.sampsize 
+        return sqrt(variance)
+    end
     if isa(design.data[!, x], CategoricalArray)
         gdf = groupby(design.data, x)
         p = combine(gdf, nrow => :counts)
         p.mean = p.counts ./ sum(p.counts)
         # variance of proportion
         p.var = design.fpc .* p.mean .* (1 .- p.mean) ./ (design.sampsize - 1)
-        p.sem = sqrt.(p.var)
+        p.se = sqrt.(p.var)
         return select(p, Not([:counts, :var]))
     end
-    return DataFrame(mean=mean(design.data[!, x]), sem=sem(x, design))
+    return DataFrame(mean=mean(design.data[!, x]), SE=se(x, design))
 end
 
+"""
```

Comment:
There's no need to repeat this documentation here. We can mention all the functionality for one method and give examples for all methods inside that same docstring. This is the custom from what I could tell. I suggest moving this example in the docstring above and remove this docstring.

Reply:
Perhaps do another PR for all the doctrings where you have left a comment after this is merged?